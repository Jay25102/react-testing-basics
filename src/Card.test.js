import { render, fireEvent, screen } from "@testing-library/react";
import Card from "./Card";

it("works when displaying a card", function() {
    const { testCard } = render(
        <Card
            caption="testing card"
            src="test4.com"
            currNum={3}
            totalNum={5}
        />
    );

    expect(
        screen.getByText("testing card")
    ).toBeInTheDocument();
    expect(
        screen.getByText("Image 3 of 5.")
    ).toBeInTheDocument();
    expect(
        testCard
    ).toMatchSnapshot();
});