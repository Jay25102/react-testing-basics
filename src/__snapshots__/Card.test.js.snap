// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works when displaying a card 1`] = `undefined`;
